test/sortitionPoolTest.js

```diff
@@ -229,18 +229,13 @@ describe("SortitionPool", () => {
 
   describe("updateOperatorStatus", () => {
     let aliceID
-    let bobID
 
     beforeEach(async () => {
       await staking.setStake(alice.address, 2000)
-      await staking.setStake(bob.address, 4000000)
       await pool.connect(owner).insertOperator(alice.address)
-      await pool.connect(owner).insertOperator(bob.address)
 
       aliceID = await pool.getOperatorID(alice.address)
-      bobID = await pool.getOperatorID(bob.address)
 
-      await staking.setStake(bob.address, 390000)
       await staking.setStake(alice.address, 1000)
 
       await helpers.time.mineBlocks(11)
```

Comment:
Is this needed?

Reply:
Removed https://github.com/keep-network/sortition-pools/pull/132/commits/f1524a44fe78726894ece27a993ec80aba5b4fbb